src/frontend/komponenter/Behandling/Inngangsvilkår/Inngangsvilkår.tsx

```diff
@@ -165,4 +143,31 @@ const VilkårStatusForAleneomsorg: React.FC<{ vurderinger: IVurdering[] }> = ({
     );
 };
 
+const NyttBarnSammePartner: React.FC<{ vilkår: IVilkår }> = ({ vilkår }) => {
```

Comment:
Lage en komponent for varje vilkår som er ansvarlig for visning och vurdering (mer tydligt ?)

Reply:
Fordel: lættare å håndtere spesial-fall som Aleneomsorg

Reply:
Ulempe: Mer komponenter ?

Reply:
Er definitivt ok med å ha flere komponenter hvis det gir oss mer fleksibilitet!